mysqld_exporter.go

```diff
@@ -52,6 +63,56 @@ func NewMySQLExporter(dsn string) *Exporter {
 			Help:      "The last scrape error status.",
 		}),
 		metrics: map[string]prometheus.Gauge{},
+		commands: prometheus.NewCounterVec(prometheus.CounterOpts{
+			Namespace: namespace,
+			Name:      "commands_total",
+			Help:      "Number of executed mysql commands.",
+		}, []string{"command"}),
+		connection_errors: prometheus.NewCounterVec(prometheus.CounterOpts{
+			Namespace: namespace,
+			Name:      "connection_errors_total",
+			Help:      "Number of mysql connection errors.",
+		}, []string{"error"}),
+		handlers: prometheus.NewCounterVec(prometheus.CounterOpts{
+			Namespace: namespace,
+			Name:      "handlers_total",
+			Help:      "Number of executed mysql handlers.",
+		}, []string{"handler"}),
+		innodb_buffer_pool: prometheus.NewUntypedVec(prometheus.UntypedOpts{
+			Namespace: namespace,
+			Name:      "innodb_buffer_pool_total",
+			Help:      "Mysql Innodb buffer pool metrics.",
+		}, []string{"metric"}),
+		innodb_data: prometheus.NewUntypedVec(prometheus.UntypedOpts{
+			Namespace: namespace,
+			Name:      "innodb_data_total",
+			Help:      "Mysql Innodb data operations.",
+		}, []string{"operation"}),
+		innodb_pages: prometheus.NewUntypedVec(prometheus.UntypedOpts{
+			Namespace: namespace,
+			Name:      "innodb_pages_total",
+			Help:      "Mysql Innodb page operations.",
+		}, []string{"operation"}),
+		innodb_rows: prometheus.NewUntypedVec(prometheus.UntypedOpts{
+			Namespace: namespace,
+			Name:      "innodb_rows_total",
+			Help:      "Mysql Innodb row operations.",
+		}, []string{"operation"}),
+		performance_schema: prometheus.NewCounterVec(prometheus.CounterOpts{
+			Namespace: namespace,
+			Name:      "performance_schema_total",
```

Comment:
As I checked the docs, these are all counters.
Here is a new sample from exporter:
```
# HELP mysql_performance_schema_total Mysql instrumentations that could not be loaded or created due to memory constraints
# TYPE mysql_performance_schema_total counter
mysql_performance_schema_total{instrumentation="accounts_lost"} 0
mysql_performance_schema_total{instrumentation="cond_classes_lost"} 0
mysql_performance_schema_total{instrumentation="cond_instances_lost"} 0
mysql_performance_schema_total{instrumentation="digest_lost"} 0
mysql_performance_schema_total{instrumentation="file_classes_lost"} 0
mysql_performance_schema_total{instrumentation="file_handles_lost"} 0
mysql_performance_schema_total{instrumentation="file_instances_lost"} 0
mysql_performance_schema_total{instrumentation="hosts_lost"} 0
mysql_performance_schema_total{instrumentation="locker_lost"} 0
mysql_performance_schema_total{instrumentation="mutex_classes_lost"} 0
mysql_performance_schema_total{instrumentation="mutex_instances_lost"} 0
mysql_performance_schema_total{instrumentation="rwlock_classes_lost"} 0
mysql_performance_schema_total{instrumentation="rwlock_instances_lost"} 0
mysql_performance_schema_total{instrumentation="session_connect_attrs_lost"} 0
mysql_performance_schema_total{instrumentation="socket_classes_lost"} 0
mysql_performance_schema_total{instrumentation="socket_instances_lost"} 0
mysql_performance_schema_total{instrumentation="stage_classes_lost"} 0
mysql_performance_schema_total{instrumentation="statement_classes_lost"} 0
mysql_performance_schema_total{instrumentation="table_handles_lost"} 0
mysql_performance_schema_total{instrumentation="table_instances_lost"} 0
mysql_performance_schema_total{instrumentation="thread_classes_lost"} 0
mysql_performance_schema_total{instrumentation="thread_instances_lost"} 0
mysql_performance_schema_total{instrumentation="users_lost"} 0
```